packages/office-ui-fabric-react/src/components/Callout/docs/CalloutOverview.md

```diff
@@ -1,5 +1,3 @@
-Callouts are a powerful way to simplify a user interface. They host tips and other information users need when they need it, with minimal effort on their part. Callouts can help you use screen space more effectively and reduce screen clutter. However, poorly designed Callouts can be annoying, distracting, unhelpful, overwhelming, or in the way.
+A callout is an anchored tip that can be used to teach people or guide them through the app without blocking them.
 
-Use a Callout for displaying additional contextual information about an item on the screen. Callouts also have a tail that identifies their source. A common use for Callout is the introduction of a new feature or capability of an app or site. Alternate usages include pairing the Callout with a button or clickable element for on-demand presentation of additional or supporting content.
-
-By default, Callouts that do not contain focusable elements (links, buttons etc) cannot gain focus when opened. For proper screen reader support, follow the non-focusable callout example, which treats the callout content like a status message.
+A real-world example can be seen in the Microsoft 365 Admin Center where a particularly complex concept is paired with the "i" icon (`iconClassNames.info`). When you hover over or select the icon, the callout appears. 
```

Comment:
Fair point. Since we don't have screenshot images supported in the documentation, we can't show what we're talking about here. Let's remove reference to the Admin Center, but keep the guidance about the info icon behavior.